orm/models_fields.go

```diff
@@ -46,9 +47,9 @@ const (
 
 // Define some logic enum
 const (
-	IsIntegerField         = ^-TypePositiveBigIntegerField >> 4 << 5
-	IsPositiveIntegerField = ^-TypePositiveBigIntegerField >> 8 << 9
-	IsRelField             = ^-RelReverseMany >> 14 << 15
+	IsIntegerField         = ^-TypePositiveBigIntegerField >> 5 << 6
+	IsPositiveIntegerField = ^-TypePositiveBigIntegerField >> 9 << 10
+	IsRelField             = ^-RelReverseMany >> 15 << 16
```

Comment:
Exactly what @lluishernandez said.
Adding `TypeTimeField` to the list of iota constants at line 28 of that same file means that those definitions needed to be changed by +1